test/helpers/kubernetes-upstream.ts

```diff
@@ -19,7 +19,7 @@ const UPSTREAM_POLLING_CONFIGURATION = {
 export async function getUpstreamResponseBody(
   relativeUrl: string,
 ): Promise<any> {
-  const url = `https://${config.INTERNAL_PROXY_CREDENTIALS}@kubernetes-upstream-int.dev.snyk.io/${relativeUrl}`;
+  const url = `https://${config.INTERNAL_PROXY_CREDENTIALS}@kubernetes-upstream-int${config.INTERNAL_PROXY_HOST}/${relativeUrl}`;
```

Comment:
what does the `config.INTERNAL_PROXY_HOST` look like? Would we need a preceding `.` for it?

Reply:
Yes, it includes trailing `.` in the env variables :)

Reply:
Should I remove and add it explicitly for easier understanding?

Reply:
maybe. It looks a bit confusing to include the `.`. 😅

Reply:
Fixed :)